modules/core/include/visp3/core/vpImagePoint.h

```diff
@@ -272,6 +272,38 @@ class VISP_EXPORT vpImagePoint
   */
   inline double get_v() const { return i; }
 
+  /*!
+   * Test if the image point is in a line represented by two image points.
+   *
+   * \param[in] start : Line start image point.
+   * \param[in] end : Line end image point.
+   * \return True if current image point is in the line. False otherwise.
+   */
+  inline bool isInLine(const vpImagePoint &start, const vpImagePoint &end) const
+  {
+    return ((end.get_j() >= start.get_j() && end.get_j() >= this->j && this->j >= start.get_j()) ||
+            (end.get_j() <= start.get_j() && end.get_j() <= this->j && this->j <= start.get_j())) &&
+           ((end.get_i() >= start.get_i() && end.get_i() >= this->i && this->i >= start.get_i()) ||
+            (end.get_i() <= start.get_i() && end.get_i() <= this->i && this->i <= start.get_i()));
+  }
+
+  /*!
+   * Return next image point of the current image point in the line [start,end].
+   *
```

Comment:
~~Return next image point of the current image point in the line [start,end].~~
Considering current image point, returns the next image point that belongs to the line [start,end].